sections/main-product.liquid

```diff
@@ -1476,12 +1476,16 @@
           "label": "t:sections.main-product.settings.gallery_layout.options__1.label"
```

Comment:
Ludo, this is really great feedback!

I feel the same way about settings in the editor. I'm hopeful for some new editor enhancement / improvements. But this could also indicate that we perhaps have _too many_ settings? _Some_ settings that aren't so clear + makes it difficult to determine where they should live -- _just thinking aloud as well_ 🤔 

And as for this PR though, placing the settings within "Product information" section was part of the original design.

Reply:
Sounds good 👌 
Yeah it makes sense to add our new settings here and maybe changing the way it's coming up in the editor would mean too many changes in the code anyway but I take a minute sometimes realizing where to go.